src/Symfony/Component/HttpFoundation/ParameterBag.php

```diff
@@ -202,4 +202,22 @@ public function count()
     {
         return \count($this->parameters);
     }
+
+    /**
+     * Returns the parameter value converted to array.
+     *
+     * @param string $key
+     * @param array $default
+     * @return array
+     */
+    public function getJsonArray($key, $default = [])
```

Comment:
missed type hint?

Reply:
can we switch to Russian, please? :)

Reply:
Вы имеете в виду про строгую типизацию аргументов?

Reply:
@SHTIKOV да, стоит указать [тайп-хинт](https://www.php.net/manual/ru/functions.arguments.php#functions.arguments.type-declaration)

@SHTIKOV yes, you should use [type-hints](https://www.php.net/manual/ru/functions.arguments.php#functions.arguments.type-declaration)
```suggestion
    public function getJsonArray(string $key, $default = [])
```

Reply:
Хорошо, спасибо, и тогда добавлю тип возравщаемых данных?

Reply:
Please, switch back to English, which is the only language we use in the Symfony repository. Thanks.

Reply:
Oh, sure, I am sorry.

I added type-hint for this method.